pkg/cfn/builder/managed_nodegroup_test.go

```diff
@@ -71,27 +71,27 @@ func TestManagedPolicyResources(t *testing.T) {
 
 	for i, tt := range iamRoleTests {
 		t.Run(fmt.Sprintf("%d: %s", i, tt.description), func(t *testing.T) {
-			assert := assert.New(t)
+			require := require.New(t)
 			clusterConfig := api.NewClusterConfig()
 
 			ng := api.NewManagedNodeGroup()
 			ng.IAM.WithAddonPolicies = tt.addons
 			ng.IAM.AttachPolicyARNs = prefixPolicies(tt.attachPolicyARNs...)
 
-			stack := NewManagedNodeGroup(clusterConfig, ng, "iam-test")
+			stack := NewManagedNodeGroup(clusterConfig, ng, nil, "iam-test")
 			err := stack.AddAllResources()
-			assert.Nil(err)
+			require.Nil(err)
```

Comment:
:hushed:  What's the difference?

Reply:
The methods in `require` stop the test (calling `FailNow`) when an assertion fails.

Reply:
ah ok so probably it should always be `require` :+1: